Gemfile

```diff
@@ -1,2 +1,3 @@
 source 'https://rubygems.org'
+gem 'webrick'
```

Comment:
Dunno if this will break the build on GitHub, we will see.

Reply:
:crossed_fingers: